plugins/copybuffer/README.md

```diff
@@ -0,0 +1,14 @@
+# `copybuffer` plugin
+
+This plugin adds a keybinding that copies the command line buffer contents to
+the system clipboard.
+
+```zsh
+plugins=(... copybuffer)
+```
+
+## Features
+
+| Keybinding | Command        | Description                                        |
```

Comment:
No need for this table and features heading. Just a brief description of the CTRL+o keybinding will do.